package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "build": "yarn codegen:generic && graph build",
     "test": "yarn codegen:generic && graph test -r",
+    "test:e2e": "cd tests/e2e && docker compose up --exit-code-from runner",
```

Comment:
This will cause the docker compose up command to exit with whatever code the runner container exits with. We can take advantage of this to determine if our tests pass in CI.

Additionally, we could expose relevant ports in the docker compose and run docker compose up without this flag so that we can interact with this setup locally. Definitely a fast follow on this work.

Reply:
🔥 🤌